website/project/metadata/schemas.py

```diff
@@ -1,17 +1,11 @@
 import os
 import json
 
+
 def ensure_schema_structure(schema):
-    if 'pages' not in schema['schema']:
-        schema['schema'] = {
-            'pages': [
-                {
-                    'id': 'page1',
-                    'title': '',
-                    'questions': schema['schema']['questions'],
-                }
-            ]
-        }
+    schema['pages'] = schema.get('pages', [])
+    schema['title'] = ' '.join(schema.get('name', '').split('_'))
+    schema['version'] = schema.get('version', 1)
```

Comment:
This function appears to ensure that the (prebuilt, unchanging) JSON schemas have all required fields.

We should not need to do this at runtime every time the OSF is loaded; the schemas should be validated and complete when they are first generated. (eg: add the fields needed when the file is first saved)

This isn't necessarily a blocker for this PR, but I'm concerned that by ignoring the absence of expected fields, this code could mask other more subtle problems with any schemas added in the future. Something to consider.